app/code/Magento/Cms/Model/BlockManagment.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Model;
+
+use Magento\Cms\Api\BlockManagementInterface;
+use Magento\Cms\Api\Data\BlockInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * @api
+ */
+
+class BlockManagment implements BlockManagementInterface
+{
+    /**
+     * @var \Magento\Cms\Model\BlockFactory
+     */
+    protected $blockFactory;
```

Comment:
According to the current [Magento Technical Guidelines](http://devdocs.magento.com/guides/v2.1/coding-standards/technical-guidelines/technical-guidelines.html) all non-public properties must be `private` to encourage composition over inheritance.